cairo-vm-cli/src/main.rs

```diff
@@ -173,20 +173,20 @@ fn run(args: impl Iterator<Item = String>) -> Result<(), Error> {
         ..Default::default()
     };
 
-    let mut cairo_runner = match {
-        if args.run_from_cairo_pie {
-            let pie = CairoPie::read_zip_file(&args.filename)?;
-            let mut hint_processor = BuiltinHintProcessor::new(
-                Default::default(),
-                RunResources::new(pie.execution_resources.n_steps),
-            );
-            cairo_run::cairo_run_pie(&pie, &cairo_run_config, &mut hint_processor)
-        } else {
-            let program_content = std::fs::read(args.filename).map_err(Error::IO)?;
-            let mut hint_processor = BuiltinHintProcessor::new_empty();
-            cairo_run::cairo_run(&program_content, &cairo_run_config, &mut hint_processor)
-        }
-    } {
+    let runner_result = if args.run_from_cairo_pie {
```

Comment:
Why are we changing this?

Reply:
Codecov was complaining about does lines, I thought it was due to the match if {...} pattern instead of using a variable.